web/js/layers/layers.sidebar.js

```diff
@@ -1,13 +1,7 @@
-/**
- * @module wv.layers
- */
-var wv = wv || {};
-wv.layers = wv.layers || {};
-
-/**
- * @class wv.layers.sidebar
- */
-wv.layers.sidebar = wv.layers.sidebar || function (models, config) {
+import $ from 'jquery';
+import util from '../util/util';
+
+export function layersSidebar(models, config) {
```

Comment:
What do you think about keeping the name `sidebar`, importing this into the `layers` parent, and attaching it to the object that exports?